src/blocks/scratch3_motion.js

```diff
@@ -258,12 +258,16 @@ class Scratch3MotionBlocks {
         util.target.setXY(util.target.x, y);
     }
 
+    roundCoord (value, places) {
+        return Math.round(value * Math.pow(10, places)) / Math.pow(10, places);
+    }
+
     getX (args, util) {
-        return util.target.x;
+        return this.roundCoord(util.target.x, 8);
```

Comment:
Applying this on get would fix projects that use setX. The precision would not be bound if you are using changeXBy command blocks. Should this rather be done in target's setXY so the stored value is the correct precision instead of the temporary value returned from getX?

Reply:
I think the concern of truncating is better placed in Target than in the command blocks.

Reply:
What I mean by changeXBy not being affected is you could have multiple changeXBy operations that change by more precision that getX returns.

I'm not fully sure my comments here are helpful.

Reply:
I get what you mean, but I'm not sure where the code for `util.target` is 😛

Reply:
Ah, found it.